app/pb_kits/playbook/pb_selectable_card/selectable_card.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbSelectableCard
+    class SelectableCard
+      include Playbook::Props
+
+      partial "pb_selectable_card/selectable_card"
+
+      prop :checked, type: Playbook::Props::Boolean,
+                  default: false
+      prop :dark, type: Playbook::Props::Boolean,
+                  default: false
+      prop :disabled, type: Playbook::Props::Boolean,
+                  default: false
+      prop :icon, type: Playbook::Props::Boolean,
+                  default: false
+      prop :multi, type: Playbook::Props::Boolean,
+                  default: true
+      prop :input_id, type: Playbook::Props::String
+      prop :name
+      prop :text
+      prop :value
+
+      
```

Comment:
Extra spaces not needed. Maybe rubocop must be configured to catch this problems.